README.md

```diff
@@ -209,6 +209,10 @@ Test events have mandatory parameters that are verified when storing an event. T
 
 ## Release notes
 
+### 5.5.0
+* Atomicity when adding new pages (`doAddPages` method in CassandraCradleStorage).
```

Comment:
Refactored `CassandraCradleStorage.doAddPages` method to execute insert and update operations atomically.